libvips/foreign/webpsave.c

```diff
@@ -1183,7 +1223,11 @@ vips_foreign_save_webp_mime_init(VipsForeignSaveWebpMime *mime)
  * Use @effort to control how much CPU time to spend attempting to
  * reduce file size. A higher value means more effort and therefore CPU time
  * should be spent. It has the range 0-6 and a default value of 4.
+ * 
+ * Use @target_size to set the desired target size in bytes.
  *
+ * Use @pass to set the number of entropy-analysis passes. It has the range 1-10.
```

Comment:
I'd also explain the default, eg. "if you set @target_size and @pass is not set, @pass defaults to 3".